test/shared_test/lib_shared_inverter_test.cpp

```diff
@@ -7,193 +7,193 @@
 
 class sharedInverterTest_lib_shared_inverter : public ::testing::Test {
 protected:
-	SharedInverter* inv;
-	sandia_inverter_t sandia;
-	double pAC = 100.;
-	double ratio= 0.;
-	double loss = 0.;
-	double e = 0.01;
+    SharedInverter* inv;
+    sandia_inverter_t sandia;
+    double pDC = 100.;
+    double ratio = 0.;
+    double loss = 0.;
+    double e = 0.01;
 public:
-	void reset() {
-		pAC = 100;
-		ratio= 1.;
-		loss = 0.;
-	}
-	void SetUp() {
+    void reset() {
+        pDC = 100;
+        ratio = 1.;
+        loss = 0.;
+    }
+    void SetUp() {
         sandia.C0 = -2.06147e-07;
         sandia.C1 = 2.7e-05;
         sandia.C2 = 0.002606;
         sandia.C3 = 0.000501;
         sandia.Paco = 59860;
-        sandia.Pdco = 61130.8;
+        sandia.Pdco = 100;
```

Comment:
Why is this value now so low? This gives a Dc to Ac efficiency of 59860/100.

Reply:
I should've changed the pDC starting value rather than the inverter rating. The sandia.Pdco value was much higher than the pDC regardless of the derate with the new approach. I will correct that

Reply:
Changed PDC to 61130.8, updated test results to reflect hypothetical system size.